oak/server/rust/oak_abi/build.rs

```diff
@@ -15,11 +15,17 @@
 //
 
 fn main() {
-    oak_utils::run_protoc_rust(protoc_rust::Args {
-        out_dir: "src/proto",
-        input: &["../../../../oak/proto/oak_api.proto"],
-        includes: &["../../../../oak/proto"],
-        customize: protoc_rust::Customize::default(),
-    })
-    .expect("protoc");
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=../../../../oak/proto/oak_api.proto");
+
+    let mut config = prost_build::Config::new();
+    config.out_dir("src/proto");
```

Comment:
Should we use "OUT_DIR" here, as per https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script ? I know we were already not following that rule, but this may be a good opportunity to fix that.

Reply:
Done. It does mean we have to include the generated file from OUT_DIR see new `proto.rs`.